internal/pkg/mapper/corefiles/local_load.go

```diff
@@ -35,6 +37,7 @@ func (m *coreFilesMapper) loadMetaFile(recipe *model.LocalLoadRecipe) error {
 func (m *coreFilesMapper) loadConfigFile(recipe *model.LocalLoadRecipe) error {
 	_, _, err := recipe.Files.
 		Load(m.state.NamingGenerator().ConfigFilePath(recipe.ObjectManifest.Path())).
+		AddMetadata(filesystem.ObjectKeyMetadata, recipe.Key()).
```

Comment:
K definicii suboru si potrebujem poznacit `object key`, ... vlastne iba na tomto mieste, pre konfiguraciu, ... ale dal som to rovno vsade.

Potom ked sa vykonava `jsonnet` kod, a najdem nejaky input, tak viem ku ktoremu objektu subor patri.